### ISSUE TITLE
\recto and \verso are very common

### ISSUE THREAD

**Issue by wujastyk** [maieul/biblatex-manuscripts-philology#22]
The macros \recto and \verso, defined in manuscripts-shared.bbx, are very generic.  They clash with another package I'm using that already defines them.  And these are very much the kind of end-user macros that an editor is likely to define for themselves.  

Should you use a package user-space prefix, like \BMPrecto, \BMPverso instead, to avoid such clashes?  (Or choose some more elegant macro name.)

Are \recto \and \verso really necessary?  You define them as r and v,  but honestly, I think most users can type this, or whatever else they want, for themselves.

**Comment by maieul** [comment]
yes, it is necessary in order to easily switch between r/v and  `\textsuperscript{r}`/`\textsuperscript{v}`
